receiver/aerospikereceiver/testdata/integration/expected.json

```
@@ -199,16 +199,373 @@
+                                                    "stringValue": "error"
+                                                }
+                                            }
+                                        ],
+                                        "startTimeUnixNano": "1655995353437739197",
+                                        "timeUnixNano": "1655995368443399008",
+                                        "asInt": "0"
+                                    },
+                                    {
+                                        "attributes": [
+                                            {
+                                                "key": "type",
+                                                "value": {
+                                                    "stringValue": "basic"
+                                                }
+                                            },
+                                            {
+                                                "key": "result",
+                                                "value": {
+                                                    "stringValue": "abort"
+                                                }
+                                            }
+                                        ],
+                                        "startTimeUnixNano": "1655995353437739197",
+                                        "timeUnixNano": "1655995368443399008",
+                                        "asInt": "0"
+                                    },
+                                    {
+                                        "attributes": [
+                                            {
+                                                "key": "type",
+                                                "value": {
+                                                    "stringValue": "udf_background"
+                                                }
+                                            },
+                                            {
+                                                "key": "result",
+                                                "value": {
+                                                    "stringValue": "error"
+                                                }
+                                            }
+                                        ],
+                                        "startTimeUnixNano": "1655995353437739197",
+                                        "timeUnixNano": "1655995368443399008",
+                                        "asInt": "0"
+                                    }
+                                ],
+                                "aggregationTemporality": "AGGREGATION_TEMPORALITY_CUMULATIVE",
+                                "isMonotonic": true
+                            }
+                        }
+                    ]
+                }
+            ]
         }
     ]
-}
+}
```

nit
```suggestion
}

```

For my benefit, was this file autogenerated? I swear we had addressed this specifically so that `golden.WriteMetrics` would add this extra line.

---

It was not, it's just the JSON-ified metrics payload sent to a development OTLP server